File: Fibonacci/Fibonacci/ViewController.m
//
//  ViewController.m
//  Fibonacci
//
//  Created by 1800 on 7/11/16.
//  Copyright © 2016 1800. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


- (IBAction)PrintNumbers:(id)sender {
    
    int i =0;
    int j =1;
    int sum = i + j;
    int temp;
    
    
    NSLog(@"Print the first value %d", i);
    NSLog(@"Print the second value %d", j);
    
    for (temp = 0; temp < 10; temp++){
        
        i = j;
        j = sum;
        
        sum = i + j;
        
            NSLog(@"The sum in sequence is %d", sum);
       
        
        }
    }


@end

File: FunFacts/FunFacts/ViewController.swift
//
//  ViewController.swift
//  FunFacts
//
//  Created by Benjamine Mutebi on 2/15/15.
//  Copyright (c) 2015 Benjamine Mutebi. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var funFactLabel: UILabel!
    @IBOutlet weak var funFactButton: UIButton!
    
     let factbook = Factbook()
     let ColorWheel = colorWheel()

    override func viewDidLoad() {
        
        
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        funFactLabel.text = factbook.randomNumber();
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

    
    @IBAction func showFunFact() {
        
        var randomColor = ColorWheel.randomColor()
        funFactButton.tintColor = randomColor;
        funFactLabel.text = factbook.randomNumber();
        view.backgroundColor = randomColor;
    }
    

}

